### ISSUE TITLE
Fail to load nuclide package. Cannot find module 'fb-appsession-observer'

### ISSUE THREAD

**Issue by makbari** [facebook/nuclide#1641]
### Issue and Steps to Reproduce
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.31.1 x64
**Electron**: 2.0.7
**OS**: Mac OS X 10.13.6
**Thrown From**: [nuclide](https://github.com/facebook/nuclide) package 0.360.0

### Stack Trace

Failed to load the nuclide package

```
At Cannot find module '../../fb-appsession-observer'

Error: Cannot find module '../../fb-appsession-observer'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:354:58)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at _fbAppsessionObserver (/packages/nuclide/pkg/nuclide-logging/lib/main.js:60:16)
    at setupLoggingService (/packages/nuclide/pkg/nuclide-logging/lib/main.js:132:71)
    at initializeLogging (/packages/nuclide/pkg/nuclide-logging/lib/main.js:125:3)
    at /packages/nuclide/modules/nuclide-commons/once.js:28:21
    at /packages/nuclide/lib/main.js:250:41)
    at /packages/nuclide/lib/main.js:585:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/nuclide/lib/main-entry.js:35:18)
    at /packages/nuclide/lib/main-entry.js:37:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:240:29)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (internal/module.js:11:18)
    at customRequire (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:96:26)
    at Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/src/package.js:782:33)
    at measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:143:22)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:88:25)
    at Package.load (/Applications/Atom.app/Contents/Resources/app/src/package.js:129:16)
    at PackageManager.loadAvailablePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:619:16)
    at PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:552:25)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:690:23)
    at /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:491:35
```

### Commands

```
     -2:55.6.0 settings-view:open (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.8 
atom-beautify 0.33.4 
atom-ternjs 0.18.3 
atom-typescript 12.6.3 
busy-signal 1.4.3 
emmet 2.4.3 
file-icons 2.1.25 
git-plus 8.1.0 
highlight-selected 0.14.0 
intentions 1.1.5 
language-babel 2.85.0 
language-graphql 0.11.0 
language-haskell 1.17.8 
language-ini 1.20.0 
language-kotlin 0.5.0 
language-lua 0.9.11 
language-ocaml 1.9.5 
language-rust 0.4.12 
language-scala 1.1.9 
language-swift 0.5.0 
language-thrift 1.0.2 
linter 2.2.0 
linter-ui-default 1.7.1 
nuclide 0.360.0 
nuclide-format-js 0.0.47 
set-syntax 0.4.0 
sort-lines 0.18.0 
tool-bar 1.1.10 
```

### Expected Behavior
Tell us what should happen.

### Actual Behavior
Tell us what happens instead.

### Versions
* Atom: 1.31.1
* Nuclide: 0.360.0
* Client OS: MAC os
* Server OS (optional):

### Additional Details
* Installed packages (`apm ls --installed`):

**Comment by qwertzguy** [comment]
Getting the same error on Windows 10 after updating to nuclide 0.360.0 and restarting Atom.

**Comment by CoolCyberBrain** [comment]
duplicate of #1640 and #1639

**Comment by CaptainMorgans** [comment]
I am also getting the same error on Windows 10. Nuclide version 0.360.0.

Error below. 


Cannot find module '../../fb-appsession-observer'
Hide Stack Trace
Error: Cannot find module '../../fb-appsession-observer'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:154364)
    at Module.require (file:///C:/Users/Stephen/AppData/Local/atom/app-1.32.0-beta3/resources/app.asar/static/index.js:40:43)
    at require (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:145671)
    at _fbAppsessionObserver (C:\Users\Stephen\.atom\packages\nuclide\pkg\nuclide-logging\lib\main.js:60:16)
    at setupLoggingService (C:\Users\Stephen\.atom\packages\nuclide\pkg\nuclide-logging\lib\main.js:132:71)
    at initializeLogging (C:\Users\Stephen\.atom\packages\nuclide\pkg\nuclide-logging\lib\main.js:125:3)
    at C:\Users\Stephen\.atom\packages\nuclide\modules\nuclide-commons\once.js:28:21
    at Object.<anonymous> (C:\Users\Stephen\.atom\packages\nuclide\lib\main.js:250:41)
    at Object.<anonymous> (C:\Users\Stephen\.atom\packages\nuclide\lib\main.js:585:3)
    at Module.get_Module._compile (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:146381)
    at Object.value [as .js] (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (file:///C:/Users/Stephen/AppData/Local/atom/app-1.32.0-beta3/resources/app.asar/static/index.js:47:45)
    at require (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:145671)
    at Object.<anonymous> (C:\Users\Stephen\.atom\packages\nuclide\lib\main-entry.js:35:18)
    at Object.<anonymous> (C:\Users\Stephen\.atom\packages\nuclide\lib\main-entry.js:37:3)
    at Module.get_Module._compile (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:146381)
    at Object.value [as .js] (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (file:///C:/Users/Stephen/AppData/Local/atom/app-1.32.0-beta3/resources/app.asar/static/index.js:47:45)
    at require (internal/module.js:11:18)
    at customRequire (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:1:622321)
    at Package.requireMainModule (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:3037220)
    at measure (C:\Users\Stephen\AppData\Local\atom\app-1.32.0-beta3\resources\app\static\<embedded>:11:3023931)

**Comment by radenkovic** [comment]
MacOS too, after update to atom@latest 🥇 
```
Error: Cannot find module '../../fb-appsession-observer'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:154364)
    at Module.require (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:145671)
    at _fbAppsessionObserver (/Users/dan/.atom/packages/nuclide/pkg/nuclide-logging/lib/main.js:60:16)
    at setupLoggingService (/Users/dan/.atom/packages/nuclide/pkg/nuclide-logging/lib/main.js:132:71)
    at initializeLogging (/Users/dan/.atom/packages/nuclide/pkg/nuclide-logging/lib/main.js:125:3)
    at /Users/dan/.atom/packages/nuclide/modules/nuclide-commons/once.js:28:21
    at Object.<anonymous> (/Users/dan/.atom/packages/nuclide/lib/main.js:250:41)
    at Object.<anonymous> (/Users/dan/.atom/packages/nuclide/lib/main.js:585:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146381)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:145671)
    at Object.<anonymous> (/Users/dan/.atom/packages/nuclide/lib/main-entry.js:35:18)
    at Object.<anonymous> (/Users/dan/.atom/packages/nuclide/lib/main-entry.js:37:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146381)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:149928)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:47:45)
    at require (internal/module.js:11:18)
    at customRequire (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:1:620607)
    at Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:2964186)
    at measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:2950897)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:2949729)
    at Package.load (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:2950470)
    at PackageManager.loadAvailablePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:377113)
    at PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:375841)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:378345)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:912430
    at r (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:907036)
    at o (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1037505)
    at ChildProcess.n.process.on.e (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1037711)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

**Comment by makbari** [comment]
Solved in Nuclide 0.362.0
